Add child features aggregation function in select by value tool (#5869)
qgis/QGIS-Documentation: docs/user_manual/working_with_vector/attribute_table.rst

if you chose the option ``Allow adding new features`` in the 
:menuselection:`Properties --> Attributes Form` menu of the airport layer.

> Child features can be used also with the :ref:`select_by_value` tool in

Reviewer: ```suggestion
Child layer can be used also with the :ref:`select_by_value` tool in
```
?

order to select features of the parent layer depending on children attributes.

> You might want to select all the regions where the *mean* altitude of the 

Reviewer: It took me some time to "understand" why people might want to do that. Maybe directly state that 
```suggestion
As an example, the :ref:`figure_select_by_value` selects all the regions where the altitude of the
```

airports is greater than 500 meters above sea level. 

> Many different aggregation functions are available in the form:

Reviewer: These are not aggregation functions, are they?

Author: actually yes. In the example you can have more airports in the same region. So you select the region where the average altitude of all the airports is greater than 500.

Reviewer: Ah OK, I did not notice that there are two levels of buttons for children

.. _figure_select_by_value:

.. figure:: img/relation_select_by_value.png

>    :align: center

Author: done

if you chose the option ``Allow adding new features`` in the 
:menuselection:`Properties --> Attributes Form` menu of the airport layer.

> Child layer can be used also with the :ref:`select_by_value` tool in
> order to select features of the parent layer depending on children attributes.

Reviewer 2: ```suggestion
The child layer can also be used in the :ref:`select_by_value` tool in
order to select features of the parent layer based on attributes of their children.
```

> As an example, the :ref:`figure_select_by_value` selects all the regions where 
> the altitude of the airports is greater than 500 meters above sea level. 

Reviewer 2: ```suggestion
In :ref:`figure_select_by_value`, all the regions where the mean altitude of
the airports is greater than 500 meters above sea level are selected. 
```

> Many different aggregation functions are available in the form:

Reviewer 2: ```suggestion
You will find that many different aggregation functions are available in the form.
```
